docs/docs/contributor-swag.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Free Swag for Gatsby Contributors!
+---
+
+Gatsby relies on contributors from the open source community, and we want to make sure you know how much it means to us that you’re willing to put time and effort into [contributing to Gatsby](/docs/how-to-contribute/).
+
+To show our appreciation, _**everyone who contributes to Gatsby is eligible to receive free swag!**_
+
+![Gatsby contributors wearing swag.](./images/gatsby-swag.jpg)
+
+### How to claim your free swag
+
+If you contribute to the Gatsby org on GitHub, you can claim your swag by [logging into the Gatsby Maintainer Dashboard](https://store.gatsbyjs.org/login) and requesting a discount code.
```

Comment:
will it be obvious how to request a discount code?

Reply:
Yep! @fk is doing a bunch of work to make sure it's super obvious.